drivers/soundwire/bus.c

```diff
@@ -1047,14 +1047,16 @@ static int sdw_initialize_slave(struct sdw_slave *slave)
 	int ret;
 	u8 val;
 
+	dev_err(slave->bus->dev,
```

Comment:
Should this be dev_dbg?

Reply:
you missed the point @bardliao. This is added on purpose to show that we program the register and not the PARITY field, but we still get parity interrupts!

Reply:
Maybe I'm also missing the point but I'm not sure either how this is useful here. I'd understand a (possibly non-repeating) `dev_err()` in the interrupt handler when such a parity interrupt is triggered, or even a `WARN_ON_ONCE()` but I'm not sure what sense it makes here.

Reply:
@lyakh When I see this trace in dmesg I can know that I do test with this PR :)